tests/templates/test_integration.py

```diff
@@ -76,18 +76,7 @@
 
 # Note that the template is called in all tests using 2 wires
 
-QUBIT_DIFFABLE_NONDIFFABLE = [(qml.templates.AmplitudeEmbedding,
-                               {'features': [1 / 2, 1 / 2, 1 / 2, 1 / 2]},
-                               {'wires': [0, 1], 'normalize': False},
-                               2),
-                              (qml.templates.AmplitudeEmbedding,
-                               {'features': [1 / 2, 1 / 2, 1 / 2, 1 / 2]},
-                               {'wires': [0, 1], 'normalize': True},
-                               2),
-                              (qml.templates.BasisEmbedding,
-                               {},
-                               {'wires': [0, 1], 'features': [1, 0]},
-                               2),
```

Comment:
Any reason you removed most of these?

Reply:
I was a bit inconsistent here...the `test_integration` file will be deleted once all refactor PRs are merged, because the old tests are superseeded. I first deleted things from this file in each PR, then realised I can just leave it and delete in the end. So I'll take care of this!